### DISCUSSION TITLE
UTM Stuck on startup.nsh

### DISCUSSION THREAD

**Discussion by ineedsomehelppls** [utmapp/UTM#6450] [UTM Help (macOS)]
![Screenshot 2024-06-30 at 1 33 57 AM](https://github.com/utmapp/UTM/assets/174205123/cc712a7c-468a-4b07-9fc1-125073f76169)
![Screenshot 2024-06-30 at 1 34 14 AM](https://github.com/utmapp/UTM/assets/174205123/3c91dc08-ff2b-4632-ba4b-332d929382a3)
I am trying to open a windows 10 instance but it is stuck on startup.nsh. idk what to do

**Comment by LHC0521** [comment]
Disabled UEFI Boot
